tls/s2n_tls13_handshake.c

```diff
@@ -85,6 +84,70 @@ int s2n_tls13_compute_shared_secret(struct s2n_connection *conn, struct s2n_blob
     return 0;
 }
 
+int s2n_tls13_compute_pq_hybrid_shared_secret(struct s2n_connection *conn, struct s2n_blob *shared_secret) {
+    notnull_check(conn);
+    notnull_check(shared_secret);
+
+    /* conn->secure.server_ecc_evp_params should be set only during a classic/non-hybrid handshake */
+    eq_check(NULL, conn->secure.server_ecc_evp_params.negotiated_curve);
+    eq_check(NULL, conn->secure.server_ecc_evp_params.evp_pkey);
```

Comment:
NP: does null_check exist?

Reply:
It does not, and I hesitate to add it because `null_check` sounds like something to guard against a value being `NULL` (which is exactly what `notnull_check` does - but I think the two would get conflated in practice). I think it's more readable to compare equality directly to `NULL`.